src/vsr/superblock_client_table.zig

```diff
@@ -45,7 +45,7 @@ pub const ClientTable = struct {
         try entries.ensureTotalCapacity(allocator, @intCast(u32, config.clients_max));
         assert(entries.capacity() >= config.clients_max);
 
-        const sorted = try allocator.alloc(*const Entry, entries.capacity());
+        const sorted = try allocator.alloc(*const Entry, config.clients_max);
```

Comment:
I think that `capacity()` could in fact exceed `clients_max` due to hash table load factor? However, this shouldn't be necessary when it comes to sorting?

Reply:
`capacity()` can exceed `clients_max`, but `clients_max` is the maximum number of entries we will store in the hash map (and thus, the maximum that will need to be sorted). So this is (potentially) a slightly tighter bound.